deps.ffmpeg/60-srt.zsh

```diff
@@ -61,16 +61,21 @@ config() {
   )
 
   case ${target} {
-    windows-x*)
+    windows-*)
       args+=(
         -DUSE_OPENSSL_PC=OFF
         -DCMAKE_CXX_FLAGS="-static-libgcc -static-libstdc++ -w -pipe -fno-semantic-interposition"
         -DCMAKE_C_FLAGS="-static-libgcc -w -pipe -fno-semantic-interposition"
-        -DCMAKE_SHARED_LINKER_FLAGS="-static-libgcc -static-libstdc++ -L${target_config[output_dir]}/lib -Wl,--exclude-libs,ALL"
         -DSSL_LIBRARY_DIRS="${target_config[output_dir]}/lib"
         -DSSL_INCLUDE_DIRS="${target_config[output_dir]}/include"
       )
 
+      if [[ ${target} == "windows-x"* ]]; then
```

Comment:
See above, use distinct `case` match for arm64 to cleanly separate architectures.

Reply:
That's a lot of duplicate codes